src/test/pybind/CMakeLists.txt

```diff
@@ -1,2 +1,4 @@
-add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
-add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
+if(NOT FREEBSD)
+  add_ceph_test(test_ceph_daemon.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_daemon.py)
+  add_ceph_test(test_ceph_argparse.py ${CMAKE_CURRENT_SOURCE_DIR}/test_ceph_argparse.py)
```

Comment:
@wjwithagen i'd recommend leave as it is instead disabling them on FreeBSD if we have not root-caused this issue. imho, a clean "make check" with all failed tests disabled is not what we are after. you can run the test using
```
cd src
PYTHONPATH=pybind /usr/local/bin/nosetests  --stop test/pybind/test_ceph_daemon.py
```